jaeger_thrift_span.go

```diff
@@ -51,10 +51,15 @@ func BuildJaegerThrift(span *Span) *j.Span {
 
 // BuildJaegerProcessThrift creates a thrift Process type.
 func BuildJaegerProcessThrift(span *Span) *j.Process {
-	return &j.Process{
-		ServiceName: span.tracer.serviceName,
-		Tags:        buildTags(span.tracer.tags),
+	return buildJaegerProcessThrift(span.tracer)
+}
+
+func buildJaegerProcessThrift(tracer *tracer) *j.Process {
```

Comment:
We can't do BuildJaegerProcessThrift(tracer *tracer) because tracer is private and certain callers don't have access to it (transport/http). I could cast it here but why bother